app/helpers/tasklist_helper.rb

```diff
@@ -46,4 +54,9 @@ def is_incomplete?(subsection, case_log, questions)
     status = get_subsection_status(subsection, case_log, questions)
     %i[not_started in_progress].include?(status)
   end
+
+  def is_started?(subsection, case_log, questions)
+    status = get_subsection_status(subsection, case_log, questions)
+    %i[in_progress completed].include?(status)
+  end
```

Comment:
Would it make sense to turn this method and the method above (is_incomplete?) into a single method and pass in the statuses we expect the subsection to have?
It might mean less duplication if we have other status combinations in the future, but it might make it less readable as `is_incomplete?` and `is_started?` are very clear. I could be swayed either way

Reply:
Think it's alright to defer that to if/when we do in the future. Think this way is quite readable for now?